ethers-core/src/types/transaction/optimism.rs

```diff
@@ -30,6 +30,7 @@ pub struct DepositTransaction {
 
     /// If true, the transaction does not interact with the L2 block gas pool.
     /// Note: boolean is disabled (enforced to be false) starting from the Regolith upgrade.
+    #[serde(rename = "isSystemTx")]
```

Comment:
out of curiousity, why do we not just use `#[serde(rename_all = "camelCase")]` at the top?

Reply:
good question, this could have been avoided by using rename_all yes